File: metrics/test/data/test_cognitive_complexity/test_breaks_in_linear_flow/data_breaks_in_linear_flow.java
public class TestClass {
    public TestClass() {

    }
    public void method1() {
        for (Integer i = 0; i < 10; i++) break;
        while (true) {
            break;
        }
         if (false)
        {
            if ("" == "a")
            {
                System.out.println("a");
            }
            else if ("" == "b")
            {
                System.out.println("b");
            }
            else 
            {
                System.out.println("");
            }
        }
        else if (true)
        {
            System.out.println("not right");
        }
        else 
        {
            System.out.println("something is wrong");
        }
        try
        {
            System.out.println("fail");
            throw new IllegalArgumentException("Failed");
        }
        catch (IllegalArgumentException ex)
        {
            System.out.println("failed");
        }
    }
}

File: metrics/test/data/test_cognitive_complexity/test_breaks_in_linear_flow/data_breaks_in_linear_flow.cs
public class TestClass
{
    public TestClass() 
    { 
        if (true) 
        {
            Console.WriteLine("hello");
        }
    }
    public void method1() 
    {
        for (int i = 0; i < 10; i++) break;
        while (true)
        {
            continue;
        }
        if (false)
        {
            if ("" == "a")
            {
                Console.WriteLine("a");
            }
            else if ("" == "b")
            {
                Console.WriteLine("b");
            }
            else 
            {
                Console.WriteLine("");
            }
        }
        else if (true)
        {
            Console.WriteLine("not right");
        }
        else 
        {
            Console.WriteLine("something is wrong");
        }
        try
        {
            Console.WriteLine("fail");
            throw new ArgumentException("failed", "");
        }
        catch (Exception ex)
        {
            Console.WriteLine("failed");
        }
    }

}